BuildingRefrigerator/BuildingRefrigerator/refrigerator.cs

```diff
@@ -0,0 +1,140 @@
+﻿namespace BuildingRefrigerator;
+
+public class refrigerator : IComparable<refrigerator?>
+{
+    static private int _uniqueIdentifierRefrigerator = 0;
+    private List<shelf?>? _shelves;
+    private string? _model;
```

Comment:
why is this optional?

Reply:
What are you asking about if it is optional?

Reply:
shouldn't this field be required to not be null?